locale/en-US.json

```diff
@@ -94,5 +94,14 @@
     "privacy": "Privacy policy",
     "welcome_message": "Welcome {name}",
     "logout": "Sign out"
+  },
+  "reviews": {
+    "rating": "Rating",
+    "no_reviews": "Be the first to rate this product:",
+    "review_count": "({count} {count, plural, =1 {Review} other {Reviews}})",
+    "button_add": "Write a Review",
+    "button_all": "All {0} Reviews",
+    "author": "by {0}",
```

Comment:
Needs to be adjusted to use parameter name instead. See the comment in the appropriate file.

Reply:
Done.